sys/analog_util/Kconfig

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config MODULE_ANALOG_UTIL
+    bool "Analog utility function interfaces"
+    depends on MODULE_PERIPH_ADC
```

Comment:
```suggestion
    depends on HAS_PERIPH_ADC
    select MODULE_PERIPH_ADC
```

Reply:
The thing is it only uses the headers from ADC and nothing to do with ADC stuff.  Should I just keep the same?

Reply:
Ok then I think it actually does not have any dependency at all right?